features/svg-opacity.yml

```diff
@@ -0,0 +1,15 @@
+name: SVG opacity
+description: The SVG opacity properties and attributes control the transparency of parts of an SVG.
+spec: https://svgwg.org/svg2-draft/render.html#ObjectAndGroupOpacityProperties
+compat_features:
+  - css.properties.fill-opacity
+  - css.properties.flood-opacity
+  - css.properties.stop-opacity
+  - css.properties.stroke-opacity
+  - svg.elements.feFlood.flood-opacity
+  - svg.elements.stop.stop-opacity
+  - svg.global_attributes.fill-opacity
+  - svg.global_attributes.flood-opacity
+  - svg.global_attributes.opacity
+  - svg.global_attributes.stroke-opacity
+group: css
```

Comment:
do we have/need an `svg` group?

Reply:
Added in https://github.com/web-platform-dx/web-features/pull/1604/commits/39c1b73dc85faa9f0654d4d64bf3e82495397235